game/outfits.rpy

```diff
@@ -1,289 +1,2058 @@
+                    # Change Natsuki out of the uniform to be removed, if she's wearing it
+                    n 1uwdaj "Oh! I totally forgot I'm wearing it already!"
+                    extend 1fslssl " Ehehe."
+
+                    play audio clothing_ruffle
+                    $ Natsuki.setOutfit(jn_outfits.get_outfit("jn_casual_clothes"))
+                    with Fade(out_time=0.1, hold_time=1, in_time=0.5, color="#181212")
+
+                n 1nchgn "Okaaay!{w=1.5}{nw}"
+                extend 1ncsbg " Just give me a second here...{w=1.5}{nw}"
+
+                if jn_outfits.delete_custom_outfit(_return):
+                    n 1fchsm "...And it's gone!"
+
+                else:
+                    n 1kllaj "...Oh."
+                    n 1klrun "Uhmm...{w=1.5}{nw}"
+                    extend 1knmpu " [player]?"
+                    n 1kllpo "I wasn't able to remove that for some reason."
+                    n 1kllss "Sorry..."
+
+            "Nevermind.":
+                n 1nnmbo "Oh."
+                n 1ullaj "Well...{w=1.5}{nw}"
+                extend 1nllca " okay then."
+
+    else:
+        n 1nnmbo "Oh.{w=1}{nw}"
+        extend 1nchgn " Well,{w=0.1} suits me!"
+
+    jump ch30_loop
+
+# Main outfit creator label
+label outfits_create_menu:
+    show natsuki idle at jn_left
+    call screen create_outfit
+
+# Headgear selection for outfit creator flow
+label outfits_create_select_headgear:
+    python:
+        unlocked_wearables = jn_outfits.JNWearable.filter_wearables(wearable_list=jn_outfits.get_all_wearables(), unlocked=True, wearable_type=jn_outfits.JNHeadgear)
+        wearable_options = []
+
+        for wearable in unlocked_wearables:
+            wearable_options.append((jn_utils.escapeRenpySubstitutionString(wearable.display_name), wearable))
+
+        wearable_options.sort(key = lambda option: option[1].display_name)
+        wearable_options.insert(0, ("No headgear", "none"))
+
+    call screen scrollable_choice_menu(wearable_options, ("Nevermind.", None))
+
+    if isinstance(_return, basestring) or isinstance(_return, jn_outfits.JNHeadgear):
+        play audio clothing_ruffle
+        python:
+            jn_outfits._changes_made = True
+            wearable_to_apply = jn_outfits.get_wearable("jn_none") if _return == "none" else _return
+            jn_outfits._PREVIEW_OUTFIT.headgear = wearable_to_apply
+            Natsuki.setOutfit(jn_outfits._PREVIEW_OUTFIT)
+
+    jump outfits_create_menu
+
+# Hairstyle selection for outfit creator flow
+label outfits_create_select_hairstyle:
+    python:
+        unlocked_wearables = jn_outfits.JNWearable.filter_wearables(wearable_list=jn_outfits.get_all_wearables(), unlocked=True, wearable_type=jn_outfits.JNHairstyle) 
+        wearable_options = []
+
+        for wearable in unlocked_wearables:
+            wearable_options.append((jn_utils.escapeRenpySubstitutionString(wearable.display_name), wearable))
```

Comment:
I this you could use list comprehension here and in similar labels below/above.

Reply:
Sure thing